datasets/newsph/newsph.py

```diff
@@ -0,0 +1,104 @@
+"""TODO(wikitext): Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+# TODO(wikitext): BibTeX citation
+_CITATION = """\
+@article{cruz2020investigating,
+  title={Investigating the True Performance of Transformers in Low-Resource Languages: A Case Study in Automatic Corpus Creation},
+  author={Jan Christian Blaise Cruz and Jose Kristian Resabal and James Lin and Dan John Velasco and Charibeth Cheng},
+  journal={arXiv preprint arXiv:2010.11574},
+  year={2020}
+}
+"""
+
+# TODO(wikitext):
+_DESCRIPTION = """\
+Large-scale dataset of Filipino news articles. Sourced for the NewsPH-NLI Project (Cruz et al., 2020).
+"""
+_URL = "https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks"
+_LICENSE = "GPL-3.0"
+_DATA_URL = "https://s3.us-east-2.amazonaws.com/blaisecruz.com/datasets/newsph"
+
+
+class NewsPHConfig(datasets.BuilderConfig):
+    """BuilderConfig for WikiText."""
+
+    def __init__(self, data_url, **kwargs):
+        """BuilderConfig for Wikitext
+
+        Args:
+          data_url: `string`, url to the dataset (word or raw level)
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(NewsPHConfig, self).__init__(
+            version=datasets.Version(
+                "1.0.0",
+            ),
+            **kwargs,
+        )
+        self.data_url = data_url
+
+
+class NewsPH(datasets.GeneratorBasedBuilder):
+    VERSION = datasets.Version("1.0.0")
+    BUILDER_CONFIGS = [
+        NewsPHConfig(
+            name="newsph",
+            data_url=_DATA_URL + "/" + "newsph.zip",
+            description=_DESCRIPTION,
+        ),
+    ]
```

Comment:
```suggestion
    ]
    BUILDER_CONFIG_CLASS = NewsPHConfig
```
we need this to specify that we're using a custom config